### PR TITLE
feat(alerts): Adds Crash Rate Alerts to Metric Alerts

### PR DESCRIPTION
**Pull request by ahmedetefy**
This PR:
- Adds the ability to create a Crash Rate alert for sessions and users
- Listens to Sessions subscription Kafka topic to receive Subscription updates for Crash Rate Alerts from snuba
- Creates an identity function that is sent as an aggregate combined with crash rate alert aggregate function to return both a crash free percentage and total session counts
- Adds a configurable minimum threshold that if set, checks for the total count in the subscription update and if greater than that value then alert comparison is dropped.

Corresponding Snuba PRs:
- Add support for sessions subscriptions 
https://github.com/getsentry/snuba/pull/2094
- Add Sessions consumer
https://github.com/getsentry/snuba/pull/2082

### CODE REVIEW

**Review by wedamija — APPROVED** [review]
I think this is your dev branch that you've been merging the other prs into, correct?

If that's the case, I'm just giving a quick review and approving so you can merge this next week. If not you might need to grab another reviewer just to be sure.

**Comment by ahmedetefy** [comment]
> I think this is your dev branch that you've been merging the other prs into, correct?
> 
> If that's the case, I'm just giving a quick review and approving so you can merge this next week. If not you might need to grab another reviewer just to be sure.

Yeah that's the one.. you have pretty much reviewed every single small PR in there :)

**Pull request merged**
